src/components/paymentTable/paymentTable.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import './paymentTable.scss';
+import '../../assets/main.css';
```

Comment:
You don't need to import it it is imported in the index.js file which going to apply to the whole app